frameworks/Python/aiohttp/app/main.py

```diff
@@ -80,14 +80,13 @@ def setup_routes(app):
         app.router.add_get('/updates/{queries:.*}', updates_raw)
 
 
-def create_app(loop):
-    app = web.Application(loop=loop)
+def create_app():
+    app = web.Application()
 
     jinja2_loader = jinja2.FileSystemLoader(str(THIS_DIR / 'templates'))
-    aiohttp_jinja2.setup(app, loader=jinja2_loader)
+    aiohttp_jinja2.setup(app, loader=jinja2_loader, enable_async=True)
```

Comment:
I thought async Jinja was very slow. Is this really a performance improvement?

Reply:
I'll revert it just in case, but I think the performance issue is only with using `await` etc. in the template.

My theory was that this might be able to improve parallelisation in future, if jinja is able to return control to the event loop while rendering or something (as this would now be doing `await render_template_async()` instead of `render_template()`). Though I don't think it has this benefit currently.

Note that async wasn't supported at all with aiohttp-jinja2<1.4.